Assets/Script/GameManager.cs

```diff
@@ -1,27 +1,34 @@
-using System.Collections;
-using System.Collections.Generic;
 using UnityEngine;
 using UnityEngine.UI;
 using System;
 
 public class GameManager : MonoBehaviour
 {
-    [SerializeField]
-    private Text PlayTime;
-    
-    private TimeSpan startTime = new TimeSpan(0, 10, 0);
-
+    public static GameManager Instance
+    {
+        get => instance ?? FindObjectOfType<GameManager>();
+    }
```

Comment:
요것도 회사에서 알려드렸는데 한줄로 가능합니다.
```suggestion
    public static GameManager Instance => instance ??= FindObjectOfType<GameManager>();
```